_2020/data-wrangling.md

```diff
@@ -240,8 +245,8 @@ wouldn't matter, but we're here to learn!
 If we wanted the _least_ common ones, we could use `head` instead of
 `tail`. There's also `sort -r`, which sorts in reverse order.
 
-Okay, so that's pretty cool, but we'd sort of like to only give the
-usernames, and maybe not one per line?
+Okay, so that's pretty cool, but what if we'd like these extract only the usernames
+as a comma-separated list instead of one per line, perhaps for a config file?
```

Comment:
I'm not sure why a "config file" comes up as an example?

Reply:
You mentioned this possibility in the lecture as a reason why one might want to do this. [Relevant moment.](https://youtu.be/sz_dsktIjt4?t=2029)